.github/workflows/lighthouse-report.yaml

```diff
@@ -0,0 +1,65 @@
+name: Lighthouse CI
+on: pull_request
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+      - name: Install and Build
+        run: |
+          yarn install
+          yarn build
+      - name: Wait for the Netlify Preview
+        uses: jakepartusch/wait-for-netlify-action@v1
+        id: netlify
+        with:
+          site_name: 'keptn-experimental-docs-site'
+      - name: Audit URLs using Lighthouse
+        id: lighthouse_audit
+        uses: treosh/lighthouse-ci-action@v3
+        with:
+          urls: |
+            https://deploy-preview-$PR_NUMBER--keptn-experimental-docs-site.netlify.app/
+            https://deploy-preview-$PR_NUMBER--keptn-experimental-docs-site.netlify.app//introduction
+          configPath: ./.github/workflows/lighthousesrc.json
+          uploadArtifacts: true
+          temporaryPublicStorage: true
+        env:
+          PR_NUMBER: ${{ github.event.pull_request.number}}
+      - name: Format lighthouse score
+        id: format_lighthouse_score
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const result = ${{ steps.lighthouse_audit.outputs.manifest }}[0].summary
+            const links = ${{ steps.lighthouse_audit.outputs.links }}
+            const formatResult = (res) => Math.round((res * 100))
+            Object.keys(result).forEach(key => result[key] = formatResult(result[key]))
+            const score = res => res >= 90 ? '🟢' : res >= 50 ? '🟠' : '🔴'
```

Comment:
can we avoid nested ternary operator here?

Reply:
Yeah, we can but I have gone through other GitHub Actions and I saw most of the people follow nested ternary operator so I just followed the same approach.

Reply:
I think this one is not difficult, but nested ternary operators are a bit hard to wrap your head around and I'd like to avoid this cognitive burden for future contributors